app.py

```diff
@@ -83,6 +83,17 @@ def add_role(email, role_name):
             logger.info("user already has role")
         db.session.commit()
 
+def add_bronzeuser(email):
```

Comment:
This function adds the user to the table but you must also add a reference to this user to the session cookie, then we can match the session cookie to the user in 'check_bronze_auth`

Reply:
you mean add another column for the session cookie?